### PR TITLE
New WordPress Apps gutenberg ref update PRs script

### PR DESCRIPTION
**Pull request by cameronvoell**
## Summary:

The `./wp_gutenberg_ref_update_prs.sh` script can be used for automatically generating Pull Requests in the [WordPress-Android](https://github.com/wordpress-mobile/WordPress-Android)and [WordPress-iOS](https://github.com/wordpress-mobile/WordPress-iOS) GitHub repositories that integrate changes from a specific [gutenberg-mobile](https://github.com/wordpress-mobile/gutenberg-mobile) branch.

## Testing
You can test this on a fork by setting `MOBILE_REPO="YOUR_GITHUB_USERNAME"` and following the rest of the instructions at the top of `./wp_gutenberg_ref_update_prs.sh`

### CODE REVIEW

**Comment by cameronvoell** [comment]
Noting for myself for later that gb-mobile bridge publish to S3 job only fires if PR is not a draft. Thinking maybe we can  remove `-draft` flag and add a "DO NOT MERGE" label for gutenberg-mobile PR.

File: wp_gutenberg_ref_update_prs.sh
Comment (cameronvoell) [inline wp_gutenberg_ref_update_prs.sh]: Noting for myself for later that gb-mobile bridge publish to S3 job only fires if PR is not a draft. Thinking maybe we can  remove -draft flag and add a "DO NOT MERGE" label for gutenberg-mobile PR.

Comment (jd-alexander) [reply wp_gutenberg_ref_update_prs.sh]: Good observation. I noticed it as well. I had made a PR ready for review just to get it to build even though I wanted it to remain as a draft.

**Review by illusaen — APPROVED** [review]
LGTM, although if you wanted to remove that draft flag and add the DNM label that sounds great too!

**Comment by AmandaRiu** [comment]
Hey @cameronvoell When I ran this script I received the following error and the script exited after 20 minutes:

Screen Shot 2021-05-12 at 5 15 45 PM

**Comment by cameronvoell** [comment]
> Hey @cameronvoell When I ran this script I received the following error and the script exited after 20 minutes:

Thanks so much for testing @AmandaRiu ! I believe that error will be fixed by my commit here: 4c8a41569adc7f83b808fea13504970a3315d4d0 . If you don't mind updating your branch and kicking it off again and letting me know if it solved that issue, would be helpful (I'm not sure how to re-create the state where cocoapod repo is out of date, since I already ran pod repo update on my local environment I cant test if the fix works :-P )

**Review by AmandaRiu — APPROVED** [review]
> Thanks so much for testing @AmandaRiu ! I believe that error will be fixed by my commit here: 4c8a415 . If you don't mind updating your branch and kicking it off again and letting me know if it solved that issue, would be helpful (I'm not sure how to re-create the state where cocoapod repo is out of date, since I already ran pod repo update on my local environment I cant test if the fix works :-P )

Tested and works beautifully @cameronvoell !

File: wp_gutenberg_ref_update_prs.sh
Comment (ceyhun) [inline wp_gutenberg_ref_update_prs.sh:145]: @cameronvoell It seemed to me like from this point the script is doing exactly the same thing with `./release_automation.sh`. Do you think it makes sense to extract common parts into some other `.sh` files? Like for example `create_wpios_pr.sh` and `create_wpandroid_pr.sh`.  I think this way we won't have to update both `./release_automation.sh` and `wp_gutenberg_ref_update_prs.sh` files when there's a change in one of the commands.

**Comment by jhnstn** [comment]
Closing since the work is stale

**Pull request closed without merging**
